packages/cli/src/zosfiles/-strings-/en.ts

```diff
@@ -468,7 +468,8 @@ export default {
             PATH: "Specifies the path where the file system is mounted." +
                 " This option and --fsname are mutually exclusive.",
             FSNAME: "Specifies the name of the mounted file system." +
-                " This option and --path are mutually exclusive."
+                " This option and --path are mutually exclusive.",
+            START: " An optional search parameter that specifies the first data set name to return in the response document."
```

Comment:
Please remove the leading whitespace from this string 🙂

Reply:
Alright, I made the change and ran the unit and integration tests successfully. I haven't made any other change. I am not sure why the Jenkins check is failing now but before it wasn't failing. I cannot see the 'Details' since I don't have access to it. Any ideas? Thanks!

Reply:
The error in Jenkins is:
`[packages/core] Changelog missing valid header. Please see CONTRIBUTING.md for changelog format.`

I don't think you should have to do anything though, since it doesn't look like you made any changes to the core SDK.